### PR TITLE
jspm is added

### PR DESCRIPTION
**Pull request by Navajyoth**
jspm is a package manager for the SystemJS universal module loader, built on top of the dynamic ES6 module loader

### CODE REVIEW

File: static/logos.json [hunk 1]
```diff
@@ -147,6 +147,9 @@
     "name": "JSX"
   },
   {
+    "name": "Jspm"
```
Comment (samiheikki) [inline static/logos.json]: Move this before JSX. Jspm comes before JSX

**Review by samiheikki — CHANGES_REQUESTED** [review]
Hi @Navajyoth Thanks for the PR 🎉 

Can you maintain the alphabetic order in logos.json (Jspm comes before JSX). After that, I can happily merge the PR 😄

**Comment by samiheikki** [comment]
This also needs rebase. Do you want to do it @Navajyoth or want me to take it from here?

File: static/logos.json (hunk 1 above)
Comment (Navajyoth) [inline static/logos.json]: I dont have permission to update issue ?. Can you make it available to me ?

**Comment by Navajyoth** [comment]
jspm alphabetical order issue fixed.

**Review by samiheikki — APPROVED** [review]

**Comment by samiheikki** [comment]
Thanks @Navajyoth 🎉

**Pull request merged**